### PR TITLE
loadtest all in one

### PR DESCRIPTION
**Pull request by syntrust**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,98 +73,104 @@ source ~/.profile
 ### Setup GoQuarkChain
 Install GoQuarkChain
```
Comment (pingke) [inline README.md]: remove Install GoQuarkChain?

File: README.md [hunk 2]
```diff
@@ -74,98 +73,104 @@ source ~/.profile
...
+is based on pre-built Docker image.
+
+### Build Cluster
+
+Build GoQuarkChain cluster executable:
 ```bash
-${USER} soft nofile ${NUMBER}
-${USER} hard nofile ${NUMBER}
+#build qkchash if not done yet:
+cd $GOPATH/src/github.com/QuarkChain/goquarkchain/consensus/qkchash/native
```
Comment (pingke) [inline README.md]: dup?

File: README.md [hunk 3]
```diff
@@ -74,98 +73,104 @@ source ~/.profile
...
-(Optional) Not needed if you are joining a testnet or mainnet. If you are starting your own network, first start the bootstrap cluster:
+If you want a cluster to be a bootstrap cluster, optionally provide a private key when you start the master service of it:
 ```bash
-cd cmd/cluser
-./cluster --cluster_config $CLUSTER_CONFIG_FILE --p2p --privkey=$BOOTSTRAP_PRIV_KEY
+cd $GOPATH/src/github.com/QuarkChain/goquarkchain/cmd/cluser
+./cluster --cluster_config $CLUSTER_CONFIG_FILE --privkey=$BOOTSTRAP_PRIV_KEY
 ```
-You can read the full bootnode URL from the console output. Then start other clusters and provide the bootnode URL.
+You can read the full boot node URL from the console output in the format: `enode://$BOOTSTRAP_PUB_KEY@$BOOTSTRAP_IP:$BOOTSTRAP_DISCOVERY_PORT`.
```
Comment (pingke) [inline README.md]: possible to add a image?

File: tests/loadtest/README.md [hunk 4]
```diff
@@ -1,36 +1,50 @@
...
+A cluster [config template file](cluster_config.json) has been provided for loadtest specifically.
 
-JSON RPCs are defined in [`rpc.proto`](../../cluster/rpc/rpc.proto). Note that there are two JSON RPC ports. By default they are 38491 for private RPCs and 38391 for public RPCs. Since you are running your own clusters you get access to both.
+Some parameters you may consider make changes:
+- `CHAIN_SIZE` defines the number of chains in the cluster, each chain has a number of shards 
+- `SHARD_SIZE` defines the number of shards per chain
+- `TARGET_BLOCK_TIME` defines the target block interval in seconds for root chain or each shard, if `POW_SIMULATE` is used for consensus     
+- `GAS_LIMIT` defines the gas limit for a block; note that in-shard transactions uses 50% of the total gas limit in a block
+- `SLAVE_LIST` defines the slaves so that the services can be located
+- `GENESIS_DIR` defines the location of loadtest.json that contains 12000 loadtest accounts
```
[the remark this answers is not in the document]
Comment (syntrust) [reply tests/loadtest/README.md]: removed

**Pull request merged**